client-app/core/composables/useErrorsTranslator.test.ts

```diff
@@ -0,0 +1,267 @@
+import { describe, expect, it, vi } from "vitest";
+import { useErrorsTranslator } from "./useErrorsTranslator";
+
+function mockI18n(): void {
+  vi.mock("vue-i18n", async (importOriginal) => {
+    const actualI18n = await importOriginal();
+    return {
+      ...{ actualI18n },
+      useI18n: vi.fn().mockReturnValue({
+        t: (key: string) => key,
+        te: (key: string) => key,
+      }),
+    };
+  });
+}
+
+describe("use-errors-translator", () => {
+  mockI18n();
+
+  const { localizedItemsErrors, setErrors } = useErrorsTranslator("validation_error");
+
+  it("Test for unexisting error", () => {
```

Comment:
naming: 
- omit "Test"
- "should return undefined for unexisting error"

name of the test case should be as if continue the phrase **use-errors-translator, it**  _should return undefined for unexisting error_